src/towncrier/test/test_build.py

```diff
@@ -407,6 +407,56 @@ def test_no_confirmation(self):
             self.assertFalse(os.path.isfile(fragment_path1))
             self.assertFalse(os.path.isfile(fragment_path2))
 
+    def test_keep_fragments(self):
+        runner = CliRunner()
```

Comment:
I think that is very important to have a docstring for the test to describe its scope and purpose

Later we might have many overlaping tests and it will be hard to know which tests is for which purpose.

```suggestion
    def test_keep_fragments(self):
        """
        The `--keep` option will build the full final news file
        without deleting the fragment files and without
        any extra CLI interaction or confirmation.
        """
        runner = CliRunner()
```